src/Online_Bookstore/Dashboard/BookDetailView.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Button,Row,Col,Container,Card,ListGroup,Form,} from "react-bootstrap";
+import {handleSubmit} from './SetReviewInLocalStorage'
+const BookDetailView = ({id,rating,booksData,setComment,comment,setRating}) => {
+    const submitClick = e=>{
+        e.preventDefault();
+        handleSubmit({id,rating,comment});
+    } 
+  return (
+    <>
+      <Container fluid>
+        <Row>
+          <Col align="center">
+            <Card className="text-center" style={{ width: "28rem" }}>
+              <Card.Img variant="top" src={booksData.image} />
+              <Card.Body>
+                <Card.Title>{booksData.title}</Card.Title>
+                <Card.Text>{booksData.desc}</Card.Text>
+              </Card.Body>
+              <ListGroup className="list-group-flush">
+                <ListGroup.Item>Author:{booksData.authors}</ListGroup.Item>
+                <ListGroup.Item>Pages:{booksData.pages}</ListGroup.Item>
+                <ListGroup.Item>Year:{booksData.year}</ListGroup.Item>
+                <ListGroup.Item>Price:{booksData.price}</ListGroup.Item>
+              </ListGroup>
+              <Card.Body>
+                <Form onSubmit={submitClick}>
+                  <Form.Group>
+                    <Form.Select
+                      aria-label="Default select example"
+                      onChange={(e) => setRating(e.target.value)}
+                    >
+                      <option>Rate the book</option>
```

Comment:
try changing the static dropdown here with a dynamic one. i.e. instead of writing every single static value in the option tag, you can create an array of numbers(as strings), and then map them here

@Poorvi-3pillar